.github/issue_template/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug bildirimi
+about: Düzeltilmesi için bug bildiriminde bulun
+title: "[BUG] - (Burayı doldurunuz)"
+labels: bug
+assignees: ''
+
+---
+
+**Bug'ın tarifi**
```

Comment:
Buralarda kalın yazı yerine header(h1 h2 gibi) kullansak daha hoş olabilir diye düşündüm sen denedin mi? :)

Reply:
Şu anki görünüm: **Bug'ın tarifi**
Dediğin görünüm: 
### Bug'ın tarifi
Bu şekil mi diyorsun @MuhammetDilmac

Reply:
Aynen abi daha hoş olur diye düşünüyorum :) Sen ne dersin @enderahmetyurt abi?